main.py

```diff
@@ -79,7 +79,30 @@ async def on_message(message: Message):
             await message.channel.send("You need to use !list first, sugar.")
             return
 
-        await command_play(message)
+        # split on spaces
+        # TODO: Possibly parse arguments before casing on command
+        command_args = message.content.split(" ")
```

Comment:
Be careful of passing a separator value to [str.split](https://docs.python.org/3/library/stdtypes.html#str.split). As the documentation states, it changes up the splitting algorithm.

Say `message.content` is `!bust<space><space>15`. If we pass `<space>` as the `sep` argument, `str.split` will actually return:

```
>>> command = "!list  15"
>>> command.split(" ")
['!list', '', '15']
```

However, if we don't set `sep`, then `str.split` will regard consecutive instances of the separator string as a single instance. Thus we end up with:

```
>>> command = "!list  15"
>>> command.split()
['!list', '15']
```

As it's likely for people to accidentally double-space, we should probably forgo setting `sep`.

```suggestion
        command_args = message.content.split()
```